source/features/comments-time-machine-links.tsx

```diff
@@ -110,7 +110,10 @@ function init(): void {
 	const commentPopupMenus = select.all('.js-reaction-popover-container ~ details:last-child:not(.rgh-time-machine-links)');
 	for (const menu of commentPopupMenus) {
 		menu.classList.add('rgh-time-machine-links');
-		const timestamp = menu.closest('.js-comment:not(.timeline-comment-group), .js-timeline-item')!.querySelector('relative-time')!.attributes.datetime.value;
+		// The timestamp of main review comments isn't in their header but in the timeline event above #5423
+		const timestamp = menu.closest('.js-comment:not([id^="pullrequestreview-"]), .js-timeline-item')!
```

Comment:
`:not(.timeline-comment-group)` was here to exclude main review comments, but now it also excludes the first comment of the issue/PR.

Fixed by making the selector more specific + added an explanatory comment for future reference.